### DISCUSSION TITLE
Faster Github Pages publishing

### DISCUSSION THREAD

**Discussion by josephrocca** [community/community#13232] [Other Feature Feedback, Questions, & Ideas]
It currently seems to take a minute or so for changes to propagate to Github Pages after making a commit. Wondering why this takes so long, and if it can be brought down to a couple of seconds? I initially thought it was just my browser cache, but it seems like it much be a CDN cache or something?

The main reason for this is that it's not a great UX to have to sit and wait for it to update when I've made a commit and I just want to check that it's all working fine.

**Comment by jfullstackdev** [comment]
even external services like the Stats will take some time to update, the official one I think is 5 minutes but can be longer

I think that will be really a heavy task for GitHub, because we all know, it was not designed to do that but simply to contain code, 

but sure, if they will be providing real-time, why not
